pegasus_wrapper/resource_request.py

```diff
@@ -85,6 +110,7 @@ class SlurmResourceRequest(ResourceRequest):
     )
     job_time_in_minutes: Optional[int] = attrib(
         validator=optional(instance_of(int)),
+        converter=default_if_none(_DEFAULT_JOB_TIME_IN_MINUTES),
```

Comment:
I am guessing this is to prevent an error on line 167. Given this change, should `job_time_in_minutes` just be `int` now? It looks like there should be no way for it to end up being `None` -- if the user passes nothing then by default it's `_DEFAULT_JOB_TIME_IN_MINUTES` and if they pass `None` explicitly then it's the same thing because of the converter.

Reply:
Good catch